### ISSUE TITLE
Accessibility: CSS :before and :after pseudo-elements insert non-decorative content [My Account]

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#51825]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

The CSS :before and :after pseudo-elements are used to add non-decorative content before and after an element's content in the document tree.

CSS :before and :after pseudo-elements specify the location of content before and after an element's document tree content. The content property, in conjunction with these pseudo-elements, specifies what is inserted.

Users who need to customize style information to view content according to their needs may not be able to access information inserted via CSS.

It is an accessibility failure to use these properties to insert non-decorative content.

![T002_ShowHideIconDefinedThroughCSS](https://github.com/user-attachments/assets/0d8bba98-1bf6-4513-ae6f-3415dce2acbe)

### Component or Page: 

Front-end Login Form on My Account Page

### Name affected element:

Show/Password icon

### Relevant code snippets:

```
<span class=""show-password-input""></span>

CSS:
.woocommerce form .show-password-input::after, .woocommerce-page form .show-password-input::after {
    font-family: WooCommerce;
    speak: never;
    font-weight: 400;
    font-variant: normal;
    text-transform: none;
    line-height: 1;
    -webkit-font-smoothing: antialiased;
    margin-left: .618em;
    content: ""\e010"";
    text-decoration: none;
}
```

### Recommended fix:

Ensure that CSS :before and :after pseudo-elements are not employed for inserting non-decorative content.

Replace this eye icon with an SVG or a font icon contained in the button rather then adding it via CSS :after.

**Comment by mikejolley** [comment]
Let's merge this with https://github.com/woocommerce/woocommerce/issues/51833 since they all share the same JavaScript.

**Comment by darrenjacoby** [comment]
@vikrampm1 this looks like a duplicate of https://github.com/woocommerce/woocommerce/issues/51834 and should fulfil these requirements once `<span>` is changed to a `<button>` element.

cc @dkotter

**Comment by vikrampm1** [comment]
@darrenjacoby please link the PR to all the similar issues when it is ready. I think that would be easier for tracking, thanks!
